pkg/storage/sqlstorage/transactions.go

```diff
@@ -40,7 +40,13 @@ func (s *Store) getTransactions(ctx context.Context, exec executor, q query.Quer
 	sb := s.transactionsQuery(q.Params)
 	sb.OrderBy("t.id desc")
 	if q.After != "" {
-		sb.Where(sb.LessThan("t.id", q.After))
+		sb.Where(sb.L("t.id", q.After))
+	}
+	if !q.StartTime.IsZero() {
```

Comment:
Maybe we should add these params on the transactionsQuery() function. This way, the "count" can match.
Otherwise the frontend cannot create a pagination while filtering with start/end dates.

Reply:
You're right, just changed it and added more tests for the `CountTransactions`.

Reply:
Nice :)